### ISSUE TITLE
DeprecationWarning from numexpr

### ISSUE THREAD

**Issue by samtygier-stfc** [mantidproject/mantidimaging#1630]
### Summary

```
/home/sam/mambaforge/envs/mantidimaging-dev/lib/python3.9/site-packages/numexpr/expressions.py:21: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    _np_version_forbids_neg_powint = LooseVersion(numpy.__version__) >= LooseVersion('1.12.0b1')
```

### Steps To Reproduce
Run pytest or start mantid imaging with warnings
`python -W default  -m mantidimaging`

### Expected Behaviour

No DeprecationWarning

### Current Behaviour

Error above

### Context

Current main

**Comment by samtygier-stfc** [comment]
This happens within tomopy. e.g.:
`python -W default  -c "import tomopy"`
because 
our environment gets an old version of numexpr 2.7.3, using 2.8 would fix this.

**Comment by samtygier-stfc** [comment]
We currently get `numexpr` because `tomopy` requires `numexpr 2.*`. Its not clear to me why `mamba install tomopy` chooses `numexpr` 2.7 instead of 2.8.

It happens with a clean up to date mamba:
```bash
docker pull condaforge/mambaforge
docker run -it --rm condaforge/mambaforge bash -c "mamba install tomopy; mamba repoquery depends tomopy; mamba list numexpr"
```

Quick fix is to add numexpr 2.8 to our requirements. But it would be good to know if this a mamba bug.

**Comment by JackEAllen** [comment]
For Windows specifically, it would seem that within tomopy `Win-ci.yml` that numexpr is pinned to 2.7.2 for tag `py38h5d928e2_2` which will affect windows conda-forge installation for CI only.  `Win-38.yml` does not pin numexpr versions. From a little exploring, I believe that the pinning of versions "may" be in relation to compatibility with older versions of MSVC, but can not be. certain on this. However, It may not be tomopy at all that is causing this issue, but rather our current version of numpy restricting the version of numexpr to 2.7.3. If setting the version of numexpr within `meta.yml` does not resolve this. It may be worth us looking into whether updating numpy does.
